cognite_toolkit/cdf_tk/run.py

```diff
@@ -177,18 +178,17 @@ def run_local_function(
     print(f"[bold]Building for environment {build_env} using {source_path!s} as sources...[/bold]")
     config.set_environment_variables()
     build_dir = tempfile.mkdtemp(prefix="build.", suffix=".tmp", dir=Path.cwd())
+
     found = False
-    for root_module, root_values in config.modules.items():
-        if not isinstance(root_values, dict):
+    for function_dir in source_path.glob("**/functions"):
```

Comment:
Is there and edge case if someone have a nested folder like this
`custom_modules/power_ops_models/data_models/views/functions/1.function_interface.view.yaml`.

Wonder if we should have similar utility function for getting resource_paths from directory.

Reply:
yes, I can take it in that task.